src/main/java/Duke.java

```diff
@@ -1,10 +1,239 @@
+import java.util.NoSuchElementException;
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+
+// A-Enums: Set Enums for Task type
+enum TASK_TYPE {
+    T,D,E
+}
+
 public class Duke {
+
+    // shows if the Duke chat bot has been activated
+    private boolean activated;
+    // Line Separator
+    private final String SEP_LINE = "____________________________________________________________";
+    // Standard boot response
+    private final String welcomeMessage =
+            SEP_LINE
+            .concat(
+            " \n____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n"
+            + "\nMade by Dr-Octavius\n")
+            .concat(SEP_LINE)
+            .concat(
+            "\nHello! I'm Duke\n"
+            + "What can I do for you?\n")
+            .concat(SEP_LINE);
+    // Level-6 -> A-Collections: Task List
+    private final List<Task> taskList = new ArrayList<>();
```

Comment:
Perhaps name this TASK_LIST, since it has been declared as a constant.

```suggestion
    private final List<Task> TASK_LIST = new ArrayList<>();
```